### ISSUE TITLE
Trying to create directory that already exists

### ISSUE THREAD

**Issue by benjaoming** [learningequality/ka-lite#3745]
After creating .deb from current develop

```
Traceback (most recent call last):
  File "/usr/share/kalite/kalitectl.py", line 645, in <module>
    manage(command, args=arguments['DJANGO_OPTIONS'])
  File "/usr/share/kalite/kalitectl.py", line 344, in manage
    utility.execute()
  File "/usr/share/kalite/python-packages/django/core/management/__init__.py", line 392, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/share/kalite/python-packages/django/core/management/__init__.py", line 263, in fetch_command
    app_name = get_commands()[subcommand]
  File "/usr/share/kalite/python-packages/django/core/management/__init__.py", line 109, in get_commands
    apps = settings.INSTALLED_APPS
  File "/usr/share/kalite/python-packages/django/conf/__init__.py", line 53, in __getattr__
    self._setup(name)
  File "/usr/share/kalite/python-packages/django/conf/__init__.py", line 48, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/share/kalite/python-packages/django/conf/__init__.py", line 132, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/share/kalite/python-packages/django/utils/importlib.py", line 35, in import_module
    __import__(name)
  File "/usr/lib/python2.7/dist-packages/kalite/settings/__init__.py", line 18, in <module>
    from .base import *
  File "/usr/lib/python2.7/dist-packages/kalite/settings/base.py", line 234, in <module>
    os.mkdir(CONTENT_ROOT)
OSError: [Errno 17] File exists: '/home/benjamin/.kalite/content'
```

**Comment by benjaoming** [comment]
Not actually sure what's going on since it checks if the path exists before trying to create it.

https://github.com/learningequality/ka-lite/commit/c85309053c726bcffef63f5641a373290d9cde2b

@jamalex any thoughts?

**Comment by benjaoming** [comment]
Doesn't happen second time I run `kalite start`. Problem might be that there are two processes doing the same thing? I dunno.. it's a little bit weird considering what the condition states...

**Comment by MCGallaspy** [comment]
Able to reproduce this error?

**Comment by benjaoming** [comment]
I'm pretty sure it was an unfortunate race-condition because two processes were loading kalite.settings in parallel.

**Comment by benjaoming** [comment]
I'm putting this as a release blocker and assigning it to myself. We need a blocking method for these kinds of automatic "initial" operations that aren't threadsafe.

**Comment by benjaoming** [comment]
@MCGallaspy this must be due to #3810 !!

**Comment by benjaoming** [comment]
There's a very increased likeliness of race conditions given that all these settings modules are imported sooooo many times!!

**Comment by benjaoming** [comment]
Will be fixed by  #3809

**Comment by rtibbles** [comment]
Assuming this is fixed as #3809 has been merged. Please correct if not @benjaoming

**Comment by benjaoming** [comment]
@rtibbles spot on, I'm 99.9% sure this was because of #3810
